### PR TITLE
tint linked contact picture in toolbar #1298

### LINKED ISSUE #1298: tint linked contact picture in toolbar
the linked contact picture as displayed in the toolbar should be tinted green. otherwise, buttons and text on it may be hidden if the picture doesn't by chance offer enough contrast

### PR DESCRIPTION
**Pull request by vhazali**
Fixes #1298

### CODE REVIEW

File: OpenKeychain/src/main/java/org/sufficientlysecure/keychain/ui/ViewKeyActivity.java [hunk 1]
```diff
@@ -923,6 +924,7 @@ protected void onPostExecute(Bitmap photo) {
                                     }
 
                                     mPhoto.setImageBitmap(photo);
+                                    mPhoto.setColorFilter(Color.argb(30, 120, 166, 69));
```
Comment (Valodim) [inline OpenKeychain/src/main/java/org/sufficientlysecure/keychain/ui/ViewKeyActivity.java]: The color should be in resources, not hardcoded. Also, please explicitly set the blending mode :+1:

**Comment by vhazali** [comment]
I'll need some more time to get this to work. Somehow I'm not able to let the correct colour show when I use the constant values in resources =\

**Comment by Valodim** [comment]
setColorFilter takes a color value, not a resource id. Both are int though, so you have to be careful with the types. You probably need something like `getResources().getColor()` there.

**Comment by vhazali** [comment]
Thanks! That solved it. I didn't know that `R.color.toolbar` will give the resource id instead of the value of the resource

**Comment by Valodim** [comment]
thanks!

**Pull request merged**